container/container.go

```diff
@@ -22,6 +28,25 @@ type Container struct {
 	sig *signature.Signature
 }
 
+// ReadFromV2 reads Container from the container.Container message.
+//
+// Does not perform if message meets NeoFS API V2 specification.
```

Comment:
Didn't catch the meaning, could you pls explain it?

Reply:
it was copied from somewhere, changed to "Does not verify if the message meets NeoFS API V2 specification."